colour/utilities/array.py

```diff
@@ -2031,6 +2031,8 @@ def tstack(
     dtype = cast(Type[DTypeFloating], optional(dtype, DEFAULT_FLOAT_DTYPE))
 
     a = as_array(a, dtype)
+    if a.ndim <= 2:
+        return a.T
```

Comment:
`np.transpose(a)` makes me happier :)

Reply:
a.T about 25% faster.  if the difference was smaller, then it would be worth it to trade off for clarity. but this function is called a lot